services/kuber/server/domain/utils/nodes/patch.go

```diff
@@ -69,18 +69,20 @@ func (p *Patcher) PatchLabels() error {
 	for _, np := range p.desiredNodepools {
 		l, err1 := nodes.GetAllLabels(np, nil)
 		if err1 != nil {
-			return fmt.Errorf("failed to create labels for %s : %w", np.Name, err)
+			return fmt.Errorf("failed to create labels for %s : %w, %w", np.Name, err, err1)
 		}
 
-		patchPath, err1 := buildJSONPatchString("replace", "/metadata/labels", l)
-		if err1 != nil {
-			return fmt.Errorf("failed to create labels patch path for %s : %w", np.Name, err)
-		}
 		for _, node := range np.Nodes {
 			nodeName := strings.TrimPrefix(node.Name, fmt.Sprintf("%s-", p.clusterID))
-			if err1 := p.kc.KubectlPatch("node", nodeName, patchPath, "--type", "json"); err1 != nil {
-				p.logger.Err(err1).Str("node", nodeName).Msgf("Failed to patch labels on node with path %s", patchPath)
-				err = fmt.Errorf("error while patching one or more nodes with labels")
+			for key, value := range l {
```

Comment:
```suggestion
			for key, value := range allLabels {
```
This just follows the previous suggestion.